src/Features/Core/Portable/UseObjectInitializer/ObjectCreationExpressionAnalyzer.cs

```diff
@@ -197,95 +186,6 @@ private bool ImplicitMemberAccessWouldBeAffected(SyntaxNode node)
 
             return false;
         }
-
-        private bool ExpressionContainsValuePattern(TExpressionSyntax expression)
```

Comment:
:memo: I'm assuming this is a trivial move to the base class, so I didn't review for preservation of details in this specific refactoring. This applies to `UseCollectionInitializer` case as well.